### ISSUE TITLE
Attributes table widget should support dynamic layers containing feature layers with definition expressions

### ISSUE THREAD

**Issue by wmbeers** [wmbeers/cmv-app#78]
See TODO statement ~line 254 of _LayerLoadMixin.js:
```
     //todo pick up subLayer definitionExpression. Not sure how this is done or if it can be done...
```

https://github.com/wmbeers/cmv-app/blob/58d2fb7d8e7d8a85cae72dcd3e715de3184b8d59/viewer/js/viewer/_LayerLoadMixin.js#L249-L257

**Comment by wmbeers** [comment]
See issue #67 for details on how to handle sub layer definition expressions. Also note that this issue is really a dependency for #67.

**Comment by wmbeers** [comment]
> See issue #67 for details on how to handle sub layer definition expressions. Also note that this issue is really a dependency for #67.

This issue was fixed in a29d9c80cd8779e87d48dbf9d94932acd8e21690, replacing the todo statement referenced in this issue's description with:

https://github.com/wmbeers/cmv-app/blob/a29d9c80cd8779e87d48dbf9d94932acd8e21690/viewer/js/viewer/_LayerLoadMixin.js#L259-L261
